src/classes/public/Context/GitHubContext/UserGitHubContext.ps1

```diff
@@ -46,8 +46,8 @@
         $this.AuthClientID = $Object.AuthClientID
         $this.DeviceFlowType = $Object.DeviceFlowType
         $this.Scope = $Object.Scope
-        $this.TokenExpirationDate = $Object.TokenExpirationDate
+        $this.TokenExpiresAt = if ($Object.TokenExpiresAt) { $Object.TokenExpiresAt } else { $Object.TokenExpirationDate }
         $this.RefreshToken = $Object.RefreshToken
-        $this.RefreshTokenExpirationDate = $Object.RefreshTokenExpirationDate
+        $this.RefreshTokenExpiresAt = if ($Object.RefreshTokenExpiresAt) { $Object.RefreshTokenExpiresAt } else { $Object.RefreshTokenExpirationDate }
```

Comment:
Dont use all the mappings. Lets only focus on the one type, as its the one we control from the function that creates the functions.